tools/testtool/main.go

```diff
@@ -0,0 +1,149 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+
+	"dagger.io/dagger"
+)
+
+func main() {
+	sandbox := flag.Bool("sandbox", false, "Run the provided command in a sandboxed environment instead of your native environment")
```

Comment:
Let's name this "incontainer" or "containerized" or something. I'm gonna object to calling a container a "sandbox" on its own